bsp/nuclei/hbird_eval/SConscript

```diff
@@ -0,0 +1,17 @@
+# for module compiling
+import os
+Import('RTT_ROOT')
+from building import *
+
+cwd = GetCurrentDir()
+objs = []
+list = os.listdir(cwd)
+
+ASFLAGS = ' -I' + cwd
```

Comment:
You can do it in a similar way in [qemu-vexpress-a9](https://github.com/RT-Thread/rt-thread/blob/master/bsp/qemu-vexpress-a9/SConstruct):
```python
env['ASCOM'] = env['ASPPCOM']
```

Reply:
Hi @BernardXiong , I checked this file https://github.com/RT-Thread/rt-thread/blob/master/bsp/qemu-vexpress-a9/SConscript, and found I need to remove `ASFLAGS = ' -I' + cwd`, do you mean that?